sretoolbox/utils/multiprocess.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2021 Red Hat
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Multiprocessing abstractions.
+"""
+
+import multiprocessing
+from multiprocessing import Pool
+
+
+def run(func, iterable, process_pool_size,
+        return_exceptions=False, **kwargs):
```

Comment:
Would it make sense to declare types here?

Reply:
makes sense but this repo does not use it yet so i will open a dedicated PR for that where we can also include mypy checks etc.